openapi.yaml

```diff
@@ -184,18 +325,37 @@ paths:
       - action
       responses:
         '200':
-          description: List of packages (datasets) including all metadata available.
+          description: 
+            en: |
+              List of packages (datasets) including all metadata available.
+            fr: |
+              Liste des paquetages (jeux de données) comprenant toutes les métadonnées disponibles. 
   "/action/package_show":
     get:
-      summary: Get metadata about one specific package (dataset)
-      description: Returns metadata about the package (dataset) corresponding to its UUID.
-        Refer to the [open data schema (type=dataset)](https://open.canada.ca/data/api/action/scheming_dataset_schema_show?type=dataset)
-        or the [open information schema (type=info)](https://open.canada.ca/data/api/action/scheming_dataset_schema_show?type=info)
-        for information about the metadata fields returned.
+      summary: 
+        en: |
+          Get metadata about one specific package (dataset)
+        fr: |
+          Obtenir les métadonnées afférentes à un paquetage particulier (jeu de données). 
+      description: 
+        en: |
+          Returns metadata about the package (dataset) corresponding to its UUID.
+          Refer to the [open data schema (type=dataset)](https://open.canada.ca/data/api/action/scheming_dataset_schema_show?type=dataset)
+          or the [open information schema (type=info)](https://open.canada.ca/data/api/action/scheming_dataset_schema_show?type=info)
+          for information about the metadata fields returned.
+        fr: |
+          Renvoie les métadonnées du paquetage (jeu de données) correspondant à son IDUU (identifiant unique universel).
+          Se référer au [schéma de données ouvert (type=jeu de données)] (https://open.canada.ca/data/api/action/scheming_dataset_schema_show?type=dataset)
```

Comment:
same issue here, can we still say `type=dataset` and provide the translation at the same time?

Reply:
Applied recommendation